tools/flang1/flang1exe/func.c

```diff
@@ -4416,6 +4416,8 @@ mk_result_sptr(int func_ast, int func_args, int *subscr, int elem_dty, int lhs,
     temp_sptr = mk_shape_sptr(shape, subscr, elem_dty);
     *retval = mk_id(temp_sptr);
     break;
+  case I_ISNAN:
+     break;
```

Comment:
Is this case necessary? If this case falls in the execution path then some uninitialized value will be returned. Else if this case is necessary can you add a test which hits this path?